components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/model/BuildTimeMetrics.java

```diff
@@ -0,0 +1,47 @@
+package com.gradle.enterprise.model;
+
+import java.time.Duration;
+
+public class BuildTimeMetrics {
+
+    public final Duration initialBuildTime;
+    public final Duration instantSavings;
+    public final Duration instantSavingsBuildTime;
+    public final Duration pendingSavings;
+    public final Duration pendingSavingsBuildTime;
```

Comment:
This class is now inconsistent with the other data classes. We should refactor the other data classes to follow this pattern as well.